modules/wechat_qrcode/test/test_qrcode.cpp

```diff
@@ -289,5 +290,37 @@ INSTANTIATE_TEST_CASE_P(/**/, Objdetect_QRCode_Monitor, testing::ValuesIn(qrcode
 INSTANTIATE_TEST_CASE_P(/**/, Objdetect_QRCode_Curved, testing::ValuesIn(qrcode_images_curved));
 // INSTANTIATE_TEST_CASE_P(/**/, Objdetect_QRCode_Multi, testing::ValuesIn(qrcode_images_multiple));
 
+TEST(Objdetect_QRCode_Big, regression) {
+    string path_detect_prototxt, path_detect_caffemodel, path_sr_prototxt, path_sr_caffemodel;
+    string model_version = "_2021-01";
+    path_detect_prototxt = findDataFile("../dnn/wechat"+model_version+"/detect.prototxt", false);
+    path_detect_caffemodel = findDataFile("../dnn/wechat"+model_version+"/detect.caffemodel", false);
+    path_sr_prototxt = findDataFile("../dnn/wechat"+model_version+"/sr.prototxt", false);
+    path_sr_caffemodel = findDataFile("../dnn/wechat"+model_version+"/sr.caffemodel", false);
```

Comment:
> `../dnn/`

Invalid path prefix.

DNN models are stored in location specified by `OPENCV_DNN_TEST_DATA_PATH`.
Don't mess them with `opencv_extra` location (and don't run "download" scripts there).

You need to use this approach: https://github.com/opencv/opencv/blame/4.6.0/modules/objdetect/test/test_main.cpp#L10-L25